vocab-builder-maven-plugin/src/main/java/com/github/tkurz/sesame/vocab/plugin/VocabularyBuilderMojo.java

```diff
@@ -245,19 +313,50 @@ public void execute() throws MojoExecutionException, MojoFailureException {
                         target = target.resolve(builder.getPackageName().replaceAll("\\.", "/"));
                         Files.createDirectories(target);
                     }
+                    if ( createUriConstants && uriGeneration != null ) {
```

Comment:
No, the global settings will be overridden with the vocabulary specific configuration - if set.